plugins/justboil.me/integratePKP.php

```diff
@@ -73,6 +73,14 @@ public function __construct() {
 		Registry::get('localeFiles', true, array($locale => array($localeFile)));
 
 		// Load user variables
+		// Set the correct default cookie path '/' -- the same as OJS core
+		// in order to get the correct session and thus user from the session
+		$configData =& Config::getData();
+		$cookiePathSet = false;
+		if (!array_key_exists('session_cookie_path', $configData['general'])) {
+			$configData['general']['session_cookie_path'] = '/';
```

Comment:
The default uses `PKPRequest::getBasePath()`, I think, not a fixed `/`. I suppose it might be necessary to wrap `basedir` around this a few times to get rid of the extra components caused by the bootstrap script being in a subdirectory.

Another note: Looking into this, I noticed another possible solution: having the JBImages bootstrap hook into `Request::getBasePath` and rewrite the result. Just posting for curiosity's sake -- no need to change to this :)

Reply:
Hmmm... The default cookie_path is = $request->getBasePath() . '/'. And my $request->getBasePath() for the cookie_path is always ''. Thus I thought it should always be '/'.
I think I do not understand what should PKPRequest::getBasePath() return here, in this case (except '')? Also how to wrap basedir (what basedir?) around to get what (''?) :-\
I thought it is functioning like this: in $request->getBasePath() the $_SERVER['SCRIPT_NAME'] is considered to get the request base path and in our case the script is in /plugins/generic/tinymce/plugins/justboil.me/ci/, differently to the index.php (when the OJS core cookie path is set). Or?

Reply:
Try putting into `/some/subdirectory/structure` instead of having it at your server root and you should see the base path get populated.

Reply:
Ahhhhhhhhh, OK, I see... :-) 👍